lua/nvim-web-devicons/icons-default.lua

```diff
@@ -3491,6 +3485,24 @@ local icons_by_operating_system = {
     cterm_color = "35",
     name = "Manjaro",
   },
+  ["mint"] = {
+    icon = "󰣭",
+    color = "#66af3d",
+    cterm_color = "70",
+    name = "Mint",
+  },
+  ["mojo"] = {
+    icon = "",
+    color = "#ff4c1f",
+    cterm_color = "196",
+    name = "Mojo",
+  },
+  ["🔥"] = {
```

Comment:
I'm not actually seeing this emoji during a hi test:
![20240607_094126](https://github.com/nvim-tree/nvim-web-devicons/assets/1505378/ce296705-89ec-448a-9d4d-b1fdd23bfe89)

Which font does this emoji come from?

Reply:
https://docs.modular.com/mojo/faq default fire emoji

Reply:
OK. I installed a font `noto-font-emoji` and can see it in browser and terminal. We'll see how it goes.

I do not think an emoji as part of a file name is a good idea; most terminals won't support it and people need to install a font to be able to see it.